imap-backup.gemspec

```diff
@@ -8,8 +9,9 @@ Gem::Specification.new do |gem|
   gem.authors       = ["Joe Yates"]
   gem.email         = ["joe.g.yates@gmail.com"]
   gem.homepage      = "https://github.com/joeyates/imap-backup"
-
-  gem.files         = `git ls-files`.split($OUTPUT_RECORD_SEPARATOR)
+  gem.licenses      = ["MIT"]
+  
+  gem.files         = Rake::FileList["**/*"].exclude(*File.read(".gitignore").split)
```

Comment:
Could this use `Dir.glob`?

Reply:
The accepted answer to [this](https://stackoverflow.com/questions/11873294/determining-the-gems-list-of-files-for-the-specification) stackoverflow question seems to list a few sensible options.

No idea if the dependency to `Rake::FileList` is something to be avoided, but I thought it kind of elegant to re-use `.gitignore` as a generic file rejection list. If you prefer `Dir.glob` I am happy to amend the commit accordingly.